File: C#/tolower.cs
using System;

public class tolower {
    public static void Main (string[] args) {
        string line = Console.ReadLine ();
        int space = line.IndexOf (' ');
        int problems = int.Parse (line.Substring (0, space));
        int cases = int.Parse (line.Substring (space + 1));
        int max = problems;
        for (int z = 0; z < problems; z++) {
            bool moreUpper = false;
            for (int x = 0; x < cases; x++) {
                string s = Console.ReadLine ();
                if (!moreUpper) {
                    for (int a = 1; a < s.Length; a++) {
                        if (char.IsUpper (s[a])) {
                            moreUpper = true;
                            break;
                        }
                    }
                }
            }
            if (moreUpper)
                max--;
        }
        Console.WriteLine (max);
    }
}

File: Java/tolower.java

// https://open.kattis.com/problems/tolower
import java.util.Scanner;

public class tolower {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int problems = sc.nextInt();
        int cases = sc.nextInt();
        int max = problems;
        sc.nextLine();
        for (int x = 0; x < problems; x++) {
            boolean moreUpper = false;
            for (int a = 0; a < cases; a++) {
                String line = sc.nextLine();
                if (!moreUpper) {
                    char[] chars = line.toCharArray();
                    for (int s = 1; s < chars.length; s++) {
                        if (Character.isUpperCase(chars[s])) {
                            moreUpper = true;
                            break;
                        }
                    }
                }
            }
            if (moreUpper)
                max--;
        }
        System.out.println(max);
        sc.close();
    }
}